pkg/kubelet/kubelet.go

```diff
@@ -1942,7 +1943,18 @@ func (kl *Kubelet) canAdmitPod(pods []*v1.Pod, pod *v1.Pod) (bool, string, strin
 	// TODO: out of resource eviction should have a pod admitter call-out
 	attrs := &lifecycle.PodAdmitAttributes{Pod: pod, OtherPods: pods}
 	for _, podAdmitHandler := range kl.admitHandlers {
+		if klog.V(4).Enabled() {
```

Comment:
This check seems redundant here as we are using `klog.V(4).InfoS` later. Please refer to [this](https://github.com/kubernetes/klog/blob/main/klog.go#L1210).

Reply:
@swatisehgal Because there is a `reflect.TypeOf` called in the log, so I use `klog.V(4).Enabled()` to reduce the performance impact on the log printing.